src/raftstore/store/peer_storage.rs

```diff
@@ -737,7 +737,7 @@ impl PeerStorage {
             region_id: self.get_region_id(),
             notifier: tx,
         };
-        if let Err(e) = self.region_sched.schedule(task) {
+        if let Err(e) = self.region_snapch.try_send(task) {
```

Comment:
Why use try_send?

Reply:
`try_send` is in retryable sender, so it functions as normal `send` in mpsc::Sender.

Reply:
pub fn try_send(&self, t: T) -> Result<(), Error> {
        self.send_with_try_times(t, 1)
}